### ISSUE TITLE
Sourcemaps don't support column/inline breakpointing

### ISSUE THREAD

**Issue by intellix** [angular/angular-cli#7307]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.

@angular/cli: 1.3.0-rc.5
node: 8.2.1
os: darwin x64
@angular/animations: 4.3.3
@angular/common: 4.3.3
@angular/compiler: 4.3.3
@angular/core: 4.3.3
@angular/forms: 4.3.3
@angular/http: 4.3.3
@angular/platform-browser: 4.3.3
@angular/platform-browser-dynamic: 4.3.3
@angular/router: 4.3.3
@angular/cli: 1.3.0-rc.5
@angular/compiler-cli: 4.3.3
@angular/language-service: 4.3.3

### Repro steps.

Add a constructor inside AppComponent and try to breakpoint inside the fat-arrow console.logs. They're visible but you can't click them.
```ts
  constructor() {
    Promise.resolve(1)
      .then(v => console.log(v))
      .then(v => console.log(v));
  }
```

If you add the same thing inside a `<script>` tag to index.html, you'll see that it'll work

### Desired functionality.

This would make debugging RxJS fat arrows a much nicer experience. Currently you have to collapse all fat-arrows to debug.

### Mention any other details that might be useful.

Screenshot of the functionality working inside a script tag:

![screenshot 2017-08-09 02 33 54](https://user-images.githubusercontent.com/1162531/29100335-3ca578e4-7cab-11e7-83c3-e9770e5777f1.png)

Kept seeing it mentioned in the new Chrome releases and could never use it for Rx: https://developers.google.com/web/updates/2017/05/devtools-release-notes#step-into-async

**Comment by asnowwolf** [comment]
I guess it may be caused by the source-map-loader. it works fine when I comment it out.

**Comment by asnowwolf** [comment]
I just tried it. I think It's resolved in cli v1.6.0. please re-verify it @intellix @filipesilva .

**Comment by intellix** [comment]
Just realised this works locally (using 1.7.1). Might have been since 1.6.0 but closing as it now works :) 
Observables are exponentially better to debug now!
